### DISCUSSION TITLE
How to make use of smtp variables

### DISCUSSION THREAD

**Discussion by ghost** [icloud-photos-downloader/icloud_photos_downloader#507] [Q&A]
Hey community,

luckily I got icloudpd set up on my Synology NAS in a container, it's syncing perfectly.
As per knowledge base articles I'm making use of variables in the Docker container config to set the apple_id, folder_path, etc. 

Now I wanted to configure SMTP to send mail notifications, but uncertain how to achieve this. Do I need to run the entire icloudpd command (which I have no awareness of as I used sync-icloud.sh and variables in my docker container) and add --smtp-username and the other required variables? How does this align with the configuration currently set by sync-icloud.sh in my container?

Guessing the answer is trivial, but can't get to it.

**Comment by AndreyNikiforov** [comment]
> Now I wanted to configure SMTP to send mail notifications, but uncertain how to achieve this. Do I need to run the entire icloudpd command (which I have no awareness of as I used sync-icloud.sh and variables in my docker container) and add --smtp-username and the other required variables? How does this align with the configuration currently set by sync-icloud.sh in my container?

I am not sure what `sync-icloud.sh` you are talking about. There is [an example script](https://github.com/icloud-photos-downloader/icloud_photos_downloader/blob/master/cron_script.sh.example) in the repo and it has nothing to do with the docker.

It may be helpful to provide some details on your set up. We can start with the cmd you use to run container -- it should give some clues at what params are you using etc. (Remove any sensitive info when sharing scripts if any).
